### PR TITLE
[APM2-717] Move OCBC Pay Anyone to be at first-level payment option

### PR DESCRIPTION
**Pull request by FhanOmise**
Move OCBC Pay Anyone out of `Mobile banking` to be at first-level payment option.

image
Screen Shot 2565-05-09 at 15 27 40

Task: https://omise.atlassian.net/browse/APM2-717

### CODE REVIEW

File: ExampleApp/Shared/PaymentSettingTableViewController.swift [hunk 1]
```diff
@@ -327,6 +327,8 @@ extension PaymentSettingTableViewController {
             return .fpx
         case rabbitLinepayCell:
             return .rabbitLinepay
+        case OCBCPAOPaymentCell:
+            return .mobileBankingOCBCPAO
```
Comment (hpariwat) [inline ExampleApp/Shared/PaymentSettingTableViewController.swift:331]: Isn't this a duplicate of line 310? 🤔

Comment (FhanOmise) [reply ExampleApp/Shared/PaymentSettingTableViewController.swift:329]: Fixed [0a6c018](https://github.com/omise/omise-ios/pull/164/commits/0a6c0187af57db1cc0116b750909323e07e6cccb)

**Review by hpariwat — APPROVED** [review]

**Review by new4762 — APPROVED** [review]

**Comment by tobias90** [comment]
@FhanOmise Can you check the text in dark mode? 
Simulator Screen Shot - iPhone 13 - 2022-05-09 at 14 00 41

**Review by jutinapas — APPROVED** [review]

File: OmiseSDK/Source.swift [hunk 2]
```diff
@@ -91,6 +91,7 @@ public struct CreateSourceParameter: Encodable {
 public enum Flow: RawRepresentable, Decodable, Equatable {
     case redirect
     case offline
+    case appredirect
```
Comment (tobias90) [inline OmiseSDK/Source.swift:94]: Since `appredirect` is made of two words, maybe we should call it `appRedirect` like we do with other enums?

**Review by tobias90 — APPROVED** [review]

**Review by tookaa2 — APPROVED** [review]

**Pull request merged**